CDCgov/prime-reportstream
# Kalish/platform/hl7 diff prototype

This PR create a command line tool that will diff two HL7 messages and print the differences in the console. 

Test Steps:
1. Pick any two files, note the changes, then run the following command and make sure it also finds those changes:
./prime hl7data -c file/path/number/1 -s /file/path/number/2

## Changes
Added a command line tool that diffs 2 hl7 files.

## Checklist

### Testing

#10795
